src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java

```diff
@@ -113,13 +165,231 @@ private void registerWebsocketMetrics() {
                 .description("Number of users connected to this Artemis instance").register(meterRegistry);
     }
 
+    private void registerExerciseAndExamMetrics() {
+        int[] ranges = { 5, 15, 30, 45, 60, 120 };
```

Comment:
```suggestion
        int[] minuteRanges = { 5, 15, 30, 45, 60, 120 };
```
Or at least a comment to specify the unit would be helpful.